data/avro-schema/src/main/resources/avro/net/corda/data/persistence/EntityResponse.avsc

```diff
@@ -11,6 +11,21 @@
         "type" : "array",
         "items": "bytes"
       }
+    },
+    {
+      "name": "size",
+      "doc": "The size of the result set.",
+      "type": "int"
+    },
+    {
+      "name": "newOffset",
+      "doc": "The new offset the result set continues from.",
+      "type": "int"
+    },
+    {
+      "name": "hasNextPage",
+      "doc": "A flag whether the result set has a next page or not.",
+      "type": "boolean"
```

Comment:
Similar comments to above.
newOffset is confusing, and reference to a "page" too.